src/components/team/styledTeam.js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+
+export const TeamWrapper = styled.section`
+    display: flex;
+    flex-direction: column;
+    width: 53.61%;
+    margin: auto;
+    padding: 150px 0;
+`
+export const TeamBlocks = styled.div`
+    display: flex;
+    justify-content: space-between;
+    margin: 80px auto 120.8px auto;
+    width: 100%;
+`
+export const TeamBlock = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`
+export const TeamImage = styled.img`
+    width: 146px;
+    height: 146px;
+    margin-bottom: 20px;
+`
+export const TeamText = styled.p`
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+`
+export const Link = styled.a`
+    width: 32px;
+    height: 32.2px;
+    background: #006699;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 21px auto 0 auto;
+    cursor: pointer;
+`
+export const LinkImg = styled.img`
+    width: 22.55px;
+    height: 21.55px;
+`
+export const PartnerRow1 = styled.div`
```

Comment:
I'm not sure how you're measuring the bottom margin here but I would double check it. I see 55px